app/utilities/PersistenceManager.java

```diff
@@ -31,6 +31,18 @@ public void createUser(User user){
         }
     }
 
+    public static void makeFavourite(User user, Restaurant restaurant){
+        for(Restaurant temp : user.getFavouriteRestaurants()) {
+            if(temp.getRestaurantId() == restaurant.getRestaurantId()) {
+                return;
```

Comment:
what should happen in this case? operation should fail with message that restaurant is already favourited